### PR TITLE
[DNM] Add HCI deployment

### PR DESCRIPTION
**Pull request by viroel**
This patch adds a new HCI deploy, with 3 (compute + ceph) external nodes.

As a pull request owner and reviewers, I checked that:
- [ ] Appropriate testing is done and actually running
- [ ] Appropriate documentation exists and/or is up-to-date:
  - [ ] README in the role
  - [ ] Content of the docs/source is reflecting the changes

Depends-On: https://github.com/openstack-k8s-operators/install_yamls/pull/447

### CODE REVIEW

File: ci_framework/roles/edpm_deploy/tasks/main.yml [hunk 1]
```diff
@@ -21,6 +21,64 @@
...
+          "{{ cifmw_edpm_deploy_env |
+             combine({
+               'EDPM_CONFIG_NET_ONLY': 'true',
+               'EDPM_STORAGE_MGMT_NETWORK': 'true',
+               'DATAPLANE_TOTAL_NODES': compute_config.amount,
+               'DATAPLANE_SINGLE_NODE': (compute_config.amount > 1) | ternary('false', 'true')
+             })
+           }}"
+        make_edpm_deploy_dryrun: "{{ cifmw_edpm_deploy_dryrun | bool }}"
+      ansible.builtin.include_role:x`x
```
Comment (psathyan) [inline ci_framework/roles/edpm_deploy/tasks/main.yml:44]: Could you check this line? It looks like there are some extra characters. ```x`x```

**Comment by viroel** [comment]
recheck

"error: You must be logged in to the server (Unauthorized)"

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

error "the server is currently unable to handle the request"

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

**Comment by viroel** [comment]
recheck

File: ci_framework/hooks/playbooks/config_ceph_backend_cinder_glance.yml [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        excludes: "kustomization.yaml"
+      register: find_result
+
+    - name: Set CR path fact
+      ansible.builtin.set_fact:
+        cifmw_edpm_kustomize_cr_path: "{{ find_result.files|map(attribute='path') | list | first }}"
+
+    - name: Kustomize Cinder configuration
+      vars:
+        cifmw_edpm_kustomize_content: |-
```
Comment (pablintino) [inline ci_framework/hooks/playbooks/config_ceph_backend_cinder_glance.yml:17]: We will need to change how `cifmw_edpm_kustomize_content` is managed, as for now, we only support one kustomization. We will need to convert this variable to a list

**Pull request closed without merging**